web/src/main.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-require('./css/style.css');
+require('./css/main.scss');
+require('./css/_popup.scss');
```

Comment:
if I use @inport instead of require, site crashes (it starts without any css)
I think it can be a webpack settings problem.
So I use require.